manager2/src/app/project/project.component.html

```diff
@@ -29,6 +29,12 @@ <h3>Request a new project creation</h3>
               <small id="nameHelp" class="form-text text-muted"><i>Optional, for information only</i></small>
           </div>
 
+          <div class="form-group row col-sm-8">
+            <label for="new_project_end">Expected end date</label>
+            <input placeholder="project end date" type="date" id="new_project_end" [ngModelOptions]="{standalone: true}" [(ngModel)]="new_project.end" class="form-control"/>
+            <small id="nameHelp" class="form-text text-muted"><i>Optional, for information only</i></small>
```

Comment:
Ah, I had not understood that. On it.

Reply:
Did that do the trick ?
I'm not sure how the security over DataTime formating works in TypeScript.
I asume we should also modify line 26 in project.service.js in /core ?
How do we allow the admin validating the request to manually modify the expiration date in case the user entered something stupid ?

Reply:
This should work, yeah.
Before 'validating' a project, an administrator can modify all fields, so we can correct any issues with provided values.